nativelib/src/main/scala/scala/scalanative/native/Atomic.scala

```diff
@@ -0,0 +1,34 @@
+package scala.scalanative
+package native
```

Comment:
This should be put in `scala.scalanative.runtime` package, it's not a public api, just an implementation detail for atomic in `java.util.concurrent`.